spark/src/main/java/org/apache/iceberg/actions/BaseSparkAction.java

```diff
@@ -128,33 +127,56 @@
     return manifestDF.union(otherMetadataFileDF).union(manifestListDF);
   }
 
+  private static boolean isExpectedCatalogLookupException(Exception exception) {
+    return exception.getMessage().contains("AnalysisException") ||
```

Comment:
Just realized I can move all of this into the Spark3Util class and just return null if we can't find it